app/views/videos/show/_video_frame.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="panel panel-default module">
+  <div class="panel-heading">
+    <b><%= video.title %></b>
+    <div id="toggle-star-<%= @video.id %>" class="pull-right">
+      <%= toggle_star_button(video) %>
+    </div>
+  </div>
+
+  <div class="h_iframe">
+      <img class="ratio" src="http://placehold.it/16x9" />
+      <iframe width="420" height="315" src="<%= video.youtube_link %>" frameborder="0" allowfullscreen></iframe>
```

Comment:
Can remove the width="420" and height="315" from the iframe because it inherits from the module.
![16x9](https://cloud.githubusercontent.com/assets/3961167/6994684/e33b5728-dad7-11e4-8339-efd77bb2f931.png)
 Consider using a transparent image rather than an external placeholder.